runestone/activecode/js/activecode.js

```diff
@@ -1272,6 +1271,19 @@ Yet another is that there is an internal error.  The internal error message is:
         }
     }
 
+    async toggleAlert(){
+        if (this.is_toggle && this.runCount == 3) {
+            if (this.errinfo != "success" || this.unit_results.substring(8,11)!=100.0){
+                setTimeout(function() {alert("Help is Available Using the Toggle Question Selector! You can try the Mixed-up Question first."); }, 500);
```

Comment:
Why the delay of 500 msec before you show the alert??

Reply:
I add this timer to show the code running results before the alert pops up, otherwise the alert will pop up immediately before seeing the actual test cases and error messages